.claude/commands/kb-status.md

```
@@ -0,0 +1,11 @@
+---
+allowed-tools:
+  - mcp__gitkb__kb_status
+  - mcp__gitkb__kb_diff
+description: Show workspace status and pending changes
+---
+
+Show the current workspace status including any uncommitted changes.
```

markdownlint-cli2 (0.20.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)